content/concepts/introduction/overview.md

```diff
@@ -1,180 +1,126 @@
+{{< /alert >}}
+
+To get started, let's begin with an overview of libp2p and its key features and
+capabilities.
+
+## A modular networking stack
+
+libp2p, (short for "library peer-to-peer", always written in lowercase as "libp2p")
+is a peer-to-peer (P2P) networking framework that enables the development
+of P2P applications. It consists of a collection of protocols, specifications, and
+libraries that facilitate P2P communication between network participants, known as
+"[peers](../fundamentals/peers.md)."
+
+P2P networks are decentralized networks in which participants communicate directly with
+one another on a relative "equal footing." No central server or
+authority controls the network, and all peers can communicate with one another
+directly. P2P networks do not require a privileged set of "servers" that behave differently
+from their "clients," as in the predominant
+[client-server model](https://en.wikipedia.org/wiki/Client%E2%80%93server_model).
+Instead, all peers in a P2P network are treated equally and can send and receive data
+to and from other peers.
+
+P2P networks can take many forms, including file-sharing systems like
+[BitTorrent](https://www.bittorrent.com/), blockchain networks like [Bitcoin](https://bitcoin.org/en/)
+and [Ethereum](https://ethereum.org/en/), and decentralized communication standards like
+[Matrix](https://matrix.org/). These systems all have different challenges and tradeoffs,
+but they share the goal of improving upon the traditional client-server networking model.
+
+libp2p was initially developed as part of the [InterPlanetary File System (IPFS)](https://ipfs.tech/)
+project as its wire protocol but has since phased out into a networking stack that has been adopted
+by a wide range of other projects as a networking layer. It provides a set of specifications that
+can be adapted to support various protocols, allowing libp2p applications to operate in diverse
+runtime and networking environments.
+
+In the context of the internet, the [OSI model](https://en.wikipedia.org/wiki/OSI_model) and the
+[TCP/IP model](https://en.wikipedia.org/wiki/Internet_protocol_suite) are conceptual models that provide
+a framework for understanding how networking protocols operate and interact. While these models offer
+a useful way to understand the functions and roles of different networking protocols, the actual
+implementations of networking protocols on the internet are often more complex and do not follow these
+models exactly.
+
+{{< alert icon="" context="info">}}
+If we consider the TCP/IP model, for instance, the primary focus of libp2p is on the
+Transport layer and the Application layer. However, these conceptual models have shortcomings.
+For example, certain tasks may be unnecessarily repeated across multiple layers, leading to
+inefficiencies. Additionally, some information may be hidden between layers, which can hinder
+opportunities for improvement.
+{{< /alert >}}
```

Comment:
```suggestion
```
I'm not sure what the intention of this paragraph is.  It mentions purported shortcomings but doesn't get specific or answer how they are addressed by libp2p

Reply:
Has been removed.